app/controllers/idam.ts

```diff
@@ -7,7 +7,7 @@ import { getIdamRedirectUrl } from '../utils/url-utils';
 
 function getLogin(req: Request, res: Response, next: NextFunction) {
   try {
-    res.redirect(paths.taskList);
+    res.redirect(paths.login);
```

Comment:
This produces an infinite loop, I think you should redirect to overview everything that was redirecting to tasklist.